src/Resources/require.js

```diff
@@ -133,12 +133,13 @@
 
     window.register = function (name, initializer) {
         if (_modules[name]) {
-            throw new ModuleRedeclareError(name);
+            throw new ModuleRedeclareError(name, _modules[name]._stack);
         }
 
         _modules[name] = {
             _initializer: initializer,
-            _module: null
+            _module: null,
+            _stack: (new TypeError).stack
```

Comment:
What is the impact on this for client side performance? I understand it's nice for debugging, but wouldn't this mean creating >100 objects for a javascript intensive app?